pyaerocom/test/test_helpers.py

```diff
@@ -125,7 +126,17 @@ def test_extract_latlon_dataarray():
     lon = [1,15, 18]
     subset = helpers.extract_latlon_dataarray(data, lat, lon, check_domain=True)
     assert isinstance(subset, xr.DataArray)
+    assert len(subset.lat) == len(lat) - 1 and len(subset.lon) == len(lon) -1
 
+@pytest.mark.parametrize('lat,lon,exception', [
+    ([], [], DataCoverageError),
+    ([1,2], [-1,2], DataCoverageError)
+    ])
+def test_extract_latlon_dataarray_no_matches(lat, lon, exception):
+    cube = helpers.make_dummy_cube_latlon(lat_res_deg=1, lon_res_deg=1, lat_range=[10,20], lon_range=[10,20])
+    data = xr.DataArray.from_iris(cube)
+    with pytest.raises(DataCoverageError):
```

Comment:
You do not use input `exception`. Shouldn't it be `pytest.raises(exception)`?

Reply:
Yes!